code/iaas/api-logic/src/main/java/io/cattle/platform/iaas/api/auth/dao/impl/AuthDaoImpl.java

```diff
@@ -39,6 +42,15 @@ public Account getAdminAccount() {
     }
 
     @Override
+    public Account getAccountById(Long id) {
+        return create()
```

Comment:
This method currently adds no value over ObjectManager.loadResource().  Being that this is auth related code, I'm assuming you should probably be adding "where state == 'active' and removed is null" to ensure that this is a valid account.  If you were to add those conditions, then it makes sense to have this method.